app/assets/stylesheets/contacts.scss

```diff
@@ -2,59 +2,22 @@
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
 // Custom bootstrap variables must be set or imported *before* bootstrap.
-#contactMap {
-  width: 100%;
-  height: 300px;
-}
-#initMapContactEdit {
-    width: 600px;
-    height: 400px;
-}
 
-.profilPic {
-  width: 200px;
-  height: 200px;
-  position: absolute;
-  top: 15%;
-  left: -10%;
-  border-radius: 50%;
-}
-
-.contactDiv {
+#contact-map {
   width: 100%;
```

Comment:
HTML class syntax is kebab-case (lowercase, with hyphens). So `contactMap` becomes -> `contact-map`